.github/workflows/bump_license_year.yml

```diff
@@ -20,14 +20,11 @@ jobs:
       - name: Checkout the latest code
         uses: actions/checkout@v3
       - name: Set Current Year
-        id: CURRENT_YEAR
         run: |
-          echo "year=$(date +'%Y')" >> "$GITHUB_OUTPUT"
+          echo "YEAR=$(date +'%Y')" >> "$GITHUB_ENV"
```

Comment:
Where is `GITHUB_ENV` going anyways? I don't see a usage here. Is this somehow connected to `env`?

Reply:
Yes it is, see https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable